tests/test_roundoff.py

```diff
@@ -0,0 +1,158 @@
+import pytest
+import numpy as np
+
+from devito import Grid, Constant, TimeFunction, Eq, Operator, switchconfig
+
+
+class TestRoundoff(object):
```

Comment:
Is it worth adding a test which tries both `safe_math=False` and `safe_math=True` with given suitable tolerances ?

Reply:
The test is explicitly designed to diverge when any floating-point error is made. Quantifying the magnitude of the error is not telling us anything additional is adds an additional layer of complexity to this test. If there is a wider accuracy issue then there are thousands of other tests already present to detect and isolate those errors.

Reply:
OK